### PR TITLE
Update .macos

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
- Update Homebrew install script
  https://brew.sh
- Install `node` via Homebrew
- Update SSH key script to latest recommendations
  https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

### CODE REVIEW

File: .macos [hunk 1]
```diff
@@ -38,10 +38,7 @@ echo "brew installing stuff"
 # bat: A cat(1) clone with syntax highlighting and Git integration.
 # switchaudio-osx: allows me to switch the audio device via the command line
 brew install git hub libdvdcss ripgrep imagemagick watchman tree bat \
-delta switchaudio-osx
-
-echo "installing node (via n-install)"
-curl -L https://git.io/n-install | bash
+delta switchaudio-osx node
```
Comment (kentcdodds) [inline .macos]: I prefer installing node this way

```suggestion
delta switchaudio-osx

echo "installing node (via n-install)"
curl -L https://git.io/n-install | bash
```

**Review by kentcdodds — COMMENTED** [review]
Hey, thanks! I want to install node via `n-install` still though.

File: .macos (hunk 1 above)
Comment (MichaelDeBoey) [reply .macos]: Any specific reason for that? 🤔

Comment (kentcdodds) [reply .macos]: I like using https://npm.im/n to manage my installed node version

**Pull request merged**